### PR TITLE
Generic: update config to Linux 5.10

### PR DESCRIPTION
**Pull request by heitbaum**

### CODE REVIEW

**Comment by chewitt** [comment]
This PR changes the default governor from ondemand to schedutil and changes an Intel audio driver. Those changes in particular need investigating.

File: projects/Generic/linux/linux.x86_64.conf [hunk 1]
```diff
@@ -2607,7 +2625,7 @@ CONFIG_RMI4_F30=y
 CONFIG_SERIO=y
 CONFIG_ARCH_MIGHT_HAVE_PC_SERIO=y
 CONFIG_SERIO_I8042=y
-CONFIG_SERIO_SERPORT=y
+CONFIG_SERIO_SERPORT=m
```
Comment (HiassofT) [inline projects/Generic/linux/linux.x86_64.conf:2628]: why did you change that from `y` to `m`?

Comment (heitbaum) [reply projects/Generic/linux/linux.x86_64.conf]: My mistake. From my build tree. Just reran from a clean upstream, and fixed. Reconfirmed the remaining is the default correct .config

**Comment by mglae** [comment]
@chewitt schedutil is now default for x86, see https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?h=v5.10&id=a00ec3874e7d326ab2dffbed92faddf6a77a84e9 and https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit?h=v5.10&id=db865272d9c4687520dc29f77e701a1b2669872f.

**Review by CvH — APPROVED** [review]
